src/webnn_native/openvino/ienn/src/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2018-2020 Intel Corporation
```

Comment:
Maybe files in this directory should be a DEPS instead of directly in the repo?

Reply:
You are right. We plan to upstream the files in this directory to [OpenVINO](https://github.com/openvinotoolkit/openvino) project. We'll add a DEPS and remove them once they land in the upstream repo. I can open a github issue in this repo to track the progress and send another PR to fix it later. Does it sound good to you?

Reply:
Sounds good.

Reply:
tracked by https://github.com/webmachinelearning/webnn-native/issues/2